src/components/WalletDropdown/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { useWeb3React } from '@web3-react/core'
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+
+import DefaultMenu from './DefaultMenu'
+import { LanguageMenu } from './LanguageMenu'
+import { TransactionHistoryMenu } from './TransactionMenu'
+
+const WalletWrapper = styled.div<{ height: string }>`
+  border-radius: 12px;
+  width: 320px;
+  height: ${(props) => props.height};
+  display: flex;
+  flex-direction: column;
+  font-size: 16px;
+  top: 60px;
+  right: 70px;
+  background-color: ${({ theme }) => theme.backgroundSurface};
+  padding-top: 16px;
```

Comment:
padding: 16px 0; idea is it makes it clear that it's supposed to be fixed between the two